index.html

```diff
@@ -84,6 +84,12 @@ <h1>GRASS GIS addons by
 </a></p>
 <!-- **** End Fork-Me-On-Gitlab-Ribbon-HTML **** -->
 
+<!-- **** Begin GRASS-overview-Ribbon-HTML. See MIT License at https://gitlab.com/seanwasere/fork-me-on-gitlab **** -->
+<p><a href="https://mundialis.github.io/grass-addons/grass_addon_overview.html">
```

Comment:
This should also work with a relative URL:
`<a href="./grass-addons/grass_addon_overview.html">` (Not tested)